armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -1131,12 +1131,6 @@ def updateNuclideTemperatures(self, blockCollectionByXsGroup=None):
     "FluxWeightedAverage": FluxWeightedAverageBlockCollection,
 }
 
-HOMOGENEOUS_BLOCK_COLLECTIONS = {
-    "Median": MedianBlockCollection,
-    "Average": AverageBlockCollection,
-    "FluxWeightedAverage": FluxWeightedAverageBlockCollection,
-}
-
```

Comment:
This is unused and just duplicate information for the normal `BLOCK_COLLECTIONS` above.